tools/update_integrity.py

```diff
@@ -0,0 +1,28 @@
+import os
+
+import click
+from registry import RegistryClient
+
+
+@click.command()
+@click.argument("module")
+@click.option("--version")
+@click.option("--registry", default=".")
+def update_integrity(module, version, registry):
+    """Update the SRI hashes in source.json of MODULE."""
+    client = RegistryClient(registry)
+    if not client.contains(module):
+        raise click.BadParameter(f"{module=} not found in registry")
+    versions = client.get_module_versions(module)
+    version = version or versions[-1][1]
```

Comment:
I noticed a potential usability issue with the default case here: If you add a new module version and forget to add the version to `metadata.json`, this will update the hashes for the latest recorded version rather than the new one.

No need to resolve this now, but if you want to go for it, we could also have this script update the version list based on a sorted `readdir`.

Reply:
added this feature